.github/workflows/lint-test.yml

```
@@ -0,0 +1,31 @@
+name: Lint, and Test
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+
+jobs:
+  lint-test:
+    name: Lint, and Test
```

Nit: The comma isn't needed here (I assume it's leftover from when this had "build" as a step as well)
```suggestion
    name: Lint and Test
```

---

fixed